README.md

```diff
@@ -379,6 +379,46 @@ WebAuthProvider.init(account)
 ```
 
 
+## Credentials Manager
+This library ships with a `CredentialsManager` class to easily store and retrieve fresh Credentials from a given `Storage`.
+
+### Usage
+1. **Instantiate the manager**
+
+```java
+AuthenticationAPIClient authClient = //create auth client
+Storage storage = //create new storage
+CredentialsManager manager = new CredentialsManager(authClient, storage);
```

Comment:
Also mention if this needs to be singleton or not, maybe a better example here on in QS